include/swift/AST/Stmt.h

```diff
@@ -802,12 +803,15 @@ class ForEachStmt : public LabeledStmt {
   Expr *Sequence;
   Expr *WhereExpr = nullptr;
   BraceStmt *Body;
-  
-  /// The iterator variable along with its initializer.
-  PatternBindingDecl *Iterator = nullptr;
-  /// The expression that advances the iterator and returns an Optional with
-  /// the next value or None to signal end-of-stream.
-  Expr *IteratorNext = nullptr;
+
+  // Set by Sema:
+  Type generatorType;
```

Comment:
'generator' is the old jargon because IteratorProtocol used to be named Generator. Can you use 'iterator' throughout?